packages/coinstac-ui/app/render/components/consortium/consortium-pipeline.jsx

```diff
@@ -1,53 +1,106 @@
-import React from 'react';
-import { Button, Col, DropdownButton, MenuItem, Row, Well } from 'react-bootstrap';
+import React, { Component } from 'react';
+import { Button, Col, DropdownButton, MenuItem, Nav, NavItem, Row, Well } from 'react-bootstrap';
 import { LinkContainer } from 'react-router-bootstrap';
+import { graphql } from 'react-apollo';
+import ApolloClient from '../../state/apollo-client';
+import PropTypes from 'prop-types';
+import { fetchAllConsortiaFunc, fetchAllPipelinesFunc, saveActivePipelineFunc } from '../../state/graphql/functions';
 
 const styles = {
   activePipelineButton: { margin: '0 5px 0 0' },
   activePipelineParagraph: { borderBottom: '1px solid black' },
   textCenter: { textAlign: 'center' },
 };
 
-const ConsortiumPipeline = () =>
-  (
-    <div>
+class ConsortiumPipeline extends Component{
+  constructor(props) {
+    super(props);
+
+    let ownedPipelines = null;
+    let sharedPipelines = null;
+    let activePipelineRef = {};
+
+    const data = ApolloClient.readQuery({ query: fetchAllPipelinesFunc });
+    ownedPipelines = data.fetchAllPipelines.filter(pipe => pipe.owningConsortium === props.consortium.id);
+    sharedPipelines = data.fetchAllPipelines.filter(pipe => pipe.shared);
+
+    if(this.props.consortium.activePipeline.length){
```

Comment:
This throws an error for me when I try to visit the page. Something about _can't read property length of null_. I don't have an `activePipeline` on my consortium object, so there's nothing to test against.

Try changing it to: `if (props.consortium.activePipeline) {` or even `if (props.consortium.activePipeline && props.consortium.activePipeline.length) {` if there are times when the field will exist but only consist of an empty string.